code/game/atoms.dm

```diff
@@ -273,10 +273,16 @@ its easier to just keep the beam vertical.
 	if(new_dir == old_dir)
 		return FALSE
 
+	for(var/i in src.contents)
```

Comment:
This seems to be a fair bit of overhead to add to a common action, what's it needed for?

Reply:
When atom turns it will call function of all atoms in its contents. In flashlight it called when mob who contains it turns, it causing flashlight to change direction.
I added it here and not in the /mob/ because it potentially could be used by every atom.